src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -55,8 +57,9 @@
     //private int autoHostile = 0;//Off
 
     //  Buildings
-    private BuildingTownHall townHall;
-    private Map<BlockPos, AbstractBuilding> buildings = new HashMap<>();
+    private BuildingTownHall                                townHall;
+    private                 Map<BlockPos, AbstractBuilding>         buildings                       = new HashMap<>();
+    private                 Map<BlockPos, Field>            fields                          = new HashMap<>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Private field 'fields' could be made final; it is only initialized in the declaration or constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3AImmutableField)